Marlin/ultralcd.cpp

```diff
@@ -958,11 +958,13 @@ void kill_screen(const char* lcd_msg) {
     }
 
     void _lcd_mesh_edit() {
+      lcdDrawUpdate = LCDVIEW_REDRAW_NOW;
       _lcd_mesh_fine_tune(PSTR("Mesh Editor"));
     }
 
     float lcd_mesh_edit() {
       lcd_goto_screen(_lcd_mesh_edit_NOP);
+      lcdDrawUpdate = LCDVIEW_REDRAW_NOW;
```

Comment:
`lcd_goto_screen` sets `lcdDrawUpdate` to `LCDVIEW_REDRAW_NEXT`, which is then converted by the drawing loop to `LCDVIEW_REDRAW_NOW`, so this line should not be needed. Did you find that this line causes the behavior to change somehow? It should have zero effect.